src/platforms/common/stm32/timing_stm32.c

```diff
@@ -44,6 +45,19 @@ static uint8_t monitor_ticks = 0;
 #define ADC_VREFINT_MIN 1404U
 #endif
 
+static void usb_config_morse_msg_update(void)
+{
+	if (usb_config_is_updated()) {
+		if (usb_config == 0) {
+			morse("NO USB HOST.", true);
+		} else {
+			morse(NULL, false);
+		}
```

Comment:
The braces on this if-else block are unnecessary, please drop them.

Reply:
Thanks for feedback, and apologies for wasting your time with formatting issues. I've made the correction in my branch.

I see clang-format v14.0 has a [style option](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#removebracesllvm) that can address this.

Reply:
It's quite ok - we need to finish writing an updated style guide, so that's on us as much as anything.

While clang-format has an option, do note the rather important warning notices on that option. The some of the bad cases indicated would actually occur in BMD which is why we have chosen not to use the option. We find it easy enough to check and indicate the problem via review rather than trying to enforce it using clang-format.